portafly/src/i18n/locales/en/product.yml

```diff
@@ -11,20 +11,27 @@ body_title_description: Body title of the page to create a new product
 
 name: Name
 name_description: Field label to show/input the name of the product.
-system_name: System name
-system_name_description: Field label to show/input the system name of the product.
+
+system_name:
+  label: System name
+  label_description: Field label to show/input the system name of the product.
+  placeholder: product-system-name
+  placeholder_description: Placeholder to show an example for the product system name.
+  helper: Use letters, numbers, dashes and underscores
+  helper_description: Helper text to explain the allowed characters for a system name.
+
 description: Description
 description_label: Field label to show/input the description of the product.
-#-------# save_button: available in shared.yml   #----PROPOSED
+#-------# create_button: available in shared.yml   #----PROPOSED
 #-------# cancel_button: available in shared.yml
 
 # Validation errors
-required_name: "<i>Name/i> is empty. You must specify a name for the product." #----PROPOSED
+required_name: "<i>Name/<i> is empty. You must specify a name for the product." #----PROPOSED
```

Comment:
Are quotes being removed?

Reply:
Yes, they are. I'll work on this.

Reply:
Since it's valid YAML anyway, I say we leave it there.